<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Экспорт в ....</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>5</y>
     <width>406</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Открытые таблицы</string>
   </property>
   <widget class="QComboBox" name="cb_opentables">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>376</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>130</y>
     <width>406</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Эксорт в </string>
   </property>
   <widget class="QLineEdit" name="ln_pathOut">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>336</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_saveas">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>15</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../../../Users/Fujitsu/.designer/backup/saveas.png</normaloff>../../../../../Users/Fujitsu/.designer/backup/saveas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>406</width>
     <height>66</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры экспорта</string>
   </property>
   <widget class="QPushButton" name="pb_coordsys">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>25</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Изменить проекцию</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_outCoordSys">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>25</y>
      <width>276</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Выходная проекция</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Проекция</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_run">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>200</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>85</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
